sylt-parser/src/parser.rs

```diff
@@ -219,6 +219,13 @@ impl<'a> Context<'a> {
         new
     }
 
+    /// Back up one token. Will not move past the beginning.
```

Comment:
```suggestion
    /// Back up one token. Will not move before the beginning.
```
Is it clearer or nah?

Reply:
I would say nah